search_service/api/document.py

```diff
@@ -89,7 +93,11 @@ def put(self) -> Tuple[Any, int]:
 
         try:
             table_dict_list = [literal_eval(table_str) for table_str in args.get('data')]
-            data, errors = self.schema(many=True).load(table_dict_list)
+            try:
+                data, errors = self.schema(many=True).load(table_dict_list), None
```

Comment:
we're already re-raising `ValidationError` later  -- no need to have `errors` now, just catch `ValidationError`, log it, and re-raise it

Reply:
this PR only upgrades dependencies and doesn't change current approach, if there is a room for refactor I would keep them as a separate one

Reply:
OK